src/styles/main.scss

```diff
@@ -1,7 +1,238 @@
-@import "utils";
-@import "fonts";
-@import "typography";
+@import "_fonts";
+@import "_header";
+@import "_footer";
+
+@mixin normalFontSize {
+  font-size: 16px;
+  line-height: 19px;
+}
 
 body {
-  background: $c-gray;
+  margin: 0;
+  font-family: "Avenir-Heavy", sans-serif;
+  color: #3d4e61;
+}
+
+//Головний блок, параметри головного блоку//
+
+.main-block {
+  display: flex;
+  justify-content: center;
+  margin-top: 29px;
+  margin-bottom: 75px;
+  &__description {
+    display: flex;
+    justify-content: space-between;
+  }
+}
+
+//блок з описом продукту//
+.product-block {
+  width: 500px;
+  display: inline-block;
+  &__article {
+    font-size: 36px;
+    line-height: 42px;
+    width: 500px;
+    height: 50px;
+  }
+  &__part-of-apartament {
+    font-family: "Avenir-Book", sans-serif;
+    @include normalFontSize;
+    color: #687c94;
+    margin-top: 10px;
+  }
+  &__parameters-apartment {
+    display: flex;
+    justify-content: start;
+  }
+  &__parameters-apartment > p {
```

Comment:
i believe that p can have a class, and try to avoid increase selectors specificity